src/components/PeopleTable.tsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import { PersonInfo } from './PersonInfo';
+import { Person } from '../types';
+
+type Props = {
+  people: Person[] | null,
+};
+
+export const PeopleTable: React.FC<Props> = ({
+  people,
+}) => {
+  return (
+    <table
+      data-cy="peopleTable"
+      className="table is-striped is-hoverable is-narrow is-fullwidth"
+    >
+      <thead>
+        <tr>
+          <th>Name</th>
+          <th>Sex</th>
+          <th>Born</th>
+          <th>Died</th>
+          <th>Mother</th>
+          <th>Father</th>
+        </tr>
+      </thead>
+
+      <tbody>
+
+        {people?.map(person => {
+          const mother = people
+            .find(p => person.motherName === p.name);
+          const father = people
+            .find(p => person.fatherName === p.name);
```

Comment:
it seems to me that these functions looks very similar let's move into separated function and reuse